src/editors/EditorContainer.tsx

```diff
@@ -38,8 +40,8 @@ const EditorContainer: React.FC<Props> = ({
         blockId={blockId}
         studioEndpointUrl={getConfig().STUDIO_BASE_URL}
         lmsEndpointUrl={getConfig().LMS_BASE_URL}
-        onClose={onClose}
-        returnFunction={returnFunction}
+        onClose={onClose ? () => onClose(location.state?.from) : null}
+        returnFunction={returnFunction ? () => returnFunction(location.state?.from) : null}
```

Comment:
See my other comment - I don't think it's necessary to pass `location.state?.from` here.

Though it may actually solve @rpenido's concern about not going back to a different website.